force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -42,7 +48,12 @@ public with sharing class FlowControlLoopsCollections {
      * @return true if the person is eligible to vote, false otherwise.
      */
     public static Boolean votingEligibility(Integer age) {
-        return null; // Replace null with the variable you used to store the result
+        if (age >= 18) {
+            return true;
+        } else {
+            return false;
+        }
+        // return null; // Replace null with the variable you used to store the result
```

Comment:
Good work. Challenge is can you be more concise here. This could be done with one line of code.